examples/run.py

```diff
@@ -24,35 +67,49 @@
     imagelab = Imagelab.load("./results", dataset_path)
     issue_types = {"light": {}, "low_information": {}}
     imagelab.find_issues(issue_types)
-    imagelab.report(issue_types.keys())
+    imagelab.report()
 
     # Check for an issue with a different threshold
     issue_types = {"dark": {"threshold": 0.2}}
     imagelab.find_issues(issue_types)
-    imagelab.report(issue_types.keys())
+    imagelab.report(issue_types=issue_types.keys())  # report specific issues
+
+    """
+    Example 3
+
+    This examples demonstrates using Imagelab to
+    1. Check for all issue types,but override the hyperparameters for an issue type
+    2. Change verbosity of report
+    3. Filter out issues occurring in more than x% of the dataset
+    4. Increase the cell size of image in image grid
+    """
 
     # Run imagelab for default issue_type, but override parameters for one or more issues
     imagelab = Imagelab(dataset_path)
     imagelab.find_issues()
+    imagelab.report(["near_duplicates"])
+
     issue_types = {"near_duplicates": {"hash_type": "phash"}}
     imagelab.find_issues(issue_types)
-    imagelab.report(issue_types.keys())
+    imagelab.report(issue_types=issue_types.keys())
+
+    # Customize report and visualize
 
-    # Customize report
     # Change verbosity
     imagelab.report(verbosity=3)
 
     # Report arg values here will overwrite verbosity defaults
     # Find top examples suffering from issues that are not present in more than 1% of the dataset
     imagelab.report(num_top_issues=5, max_prevalence=0.01)
 
-    # Visualize
-    imagelab.visualize(["light"], examples_per_issue=8, cell_size=(3, 3))
+    # Increase cell_size in the grid
+    imagelab.visualize(issue_types=["light"], examples_per_issue=8, cell_size=(3, 3))
 
-    # Get stats
-    stats = imagelab.info["statistics"]
-    print(stats.keys())
+    """
+    Example 4
 
+    This example demonstrates creating your own custom issue and using Imagelab to check for the added issue type
```

Comment:
```suggestion
    This example demonstrates creating your own custom issue and using Imagelab to detect this additional issue type, along with the default set of issues
```

Reply:
I know this ^ is not what your original code below did, but that is what I'm suggesting it should do. That is the true power of the Imagelab, not just detecting one custom issue type, but detecting it together with the built-in types